website/docs/fr-FR/container.md

```diff
@@ -168,11 +168,13 @@ Ces composants utilisent flexbox, assurez vous que le navigateur supporte cette
     <el-header style="text-align: right; font-size: 12px">
       <el-dropdown>
         <i class="el-icon-setting" style="margin-right: 15px"></i>
-        <el-dropdown-menu slot="dropdown">
-          <el-dropdown-item>View</el-dropdown-item>
-          <el-dropdown-item>Add</el-dropdown-item>
-          <el-dropdown-item>Delete</el-dropdown-item>
-        </el-dropdown-menu>
+        <template #dropdown>
+          <el-dropdown-menu>
+              <el-dropdown-item>查看</el-dropdown-item>
+              <el-dropdown-item>新增</el-dropdown-item>
+              <el-dropdown-item>删除</el-dropdown-item>
```

Comment:
You need to restore the text here.

Reply:
I'm sorry, I'll add commit.

Reply:
Don't be sorry for things that's not even a fault. Just revert the change it'll be fine